alibi/tests/test_saving.py

```diff
@@ -340,6 +341,7 @@ def test_save_AnchorImage(ai_explainer, mnist_predictor):
         assert exp0.meta == exp1.meta
 
 
+@pytest.mark.skipif(sys.platform.startswith("win"), reason="Test fails intermittently on windows platform")
```

Comment:
Is this a separate issue? If so, perhaps best to do this in a separate PR so that we can easily revert once fixed.

Reply:
Done in #812